python/cudf/cudf/core/frame.py

```diff
@@ -634,46 +633,16 @@ def _merge(
 
         result = []
         cat_codes = []
-        if sort:
-            # Pandas lexicographically sort is NOT a sort of all columns.
-            # Instead, it sorts columns in lhs, then in "on", and then rhs.
```

Comment:
Can you add back in this comment or a comment describing the behavior of how it chooses the order in which to sort columns?